src/status_im/ui/screens/home/views.cljs

```diff
@@ -187,15 +184,12 @@
                                       :animating true}]
            [react/view {:flex 1}
             [filter.views/search-input-wrapper search-filter]
-            (if (and (not search-filter)
-                     (empty? all-home-items))
-              [home-tooltip-view true hide-home-tooltip?]
-              [home-items-view
-               search-filter
-               chats
-               all-home-items
-               hide-home-tooltip?
-               filter.views/search-input-state])])]
+            [home-items-view
```

Comment:
Not introduced here, but wondering why we pass 4 params that are currently not used.